.github/workflows/ci.yml

```diff
@@ -9,15 +9,15 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v2
 
       - name: Lint and format CSS
         run: |
           npm ci
           npm run lint
 
       - name: Install PHP
-        uses: shivammathur/setup-php@2.15.0
+        uses: shivammathur/setup-php@2
```

Comment:
Meh, why does it work for one and not the other?

Reply:
https://github.com/actions/checkout's releases use a `v` prefix for tags, whereas https://github.com/shivammathur/setup-php doesn't. This is a bug in GitHub Actions itself :slightly_frowning_face:

Reply:
Yeah adding the `v` fixed it.